scripts/arangodb-docker-container-setup/docker-container-scripts/js/setup.js

```diff
@@ -28,18 +28,42 @@ else {
 
 const testData = require(process.env.ARANGODB_DESTINATION_CONTAINER_DOCKER_SHARED_VOLUME_SCRIPTS_PATH + '/test-data/testData.json');
 
-print(typeof testData.resources);
+// Could we validate these collection names (or retrieve them) from ArangoCollectionId.ts?
```

Comment:
Actually, this might be a good reason to do it within the back-end codebase. Can you remove "Could we" and replace it with a 
``` ts
// TODO [https://www.pivotaltracker.com/story/show/182132515]
```

Reply:
Done